### PR TITLE
added Khmer locale support

### PR DESCRIPTION
**Pull request by SuuSoJeat**

### CODE REVIEW

**Comment by SuuSoJeat** [comment]
How can adding 6 locale files cause the build to fail? 
Urgh! I'm frustrated.

**Comment by eldadfux** [comment]
Maybe you cloned a broken version from earlier today. Try to pull from master again.

**Comment by eldadfux** [comment]
Might also be something to do with issues in TravisCI in the last hours: https://www.traviscistatus.com/

File: app/config/locales/km.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'account.emails.verification.title' => 'ការផ្ទៀងផ្ទាត់គណនី',
+    'account.emails.verification.body' => 'en.email.auth.confirm.tpl',
+    'account.emails.recovery.title' => 'កំណត់ពាក្យសម្ងាត់ឡើងវិញ',
+    'account.emails.recovery.body' => 'en.email.auth.recovery.tpl',
+    'account.emails.invitation.title' => 'ការអញ្ជើញចូលក្នុងក្រុម %s នៅ %s',
+    'account.emails.invitation.body' => 'en.email.auth.invitation.tpl',
+
+    'locale.country.unknown' => 'មិនស្គាល់',
+
+    'countries' => include 'en.countries.php',
```
Comment (eldadfux) [inline app/config/locales/km.php]: This should be km.countries.php

File: app/config/locales/km.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'continents' => include 'en.continents.php',
```
Comment (eldadfux) [inline app/config/locales/km.php]: This should be km.continents.php

File: app/config/locales/km.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'settings.inspire' => '"សិល្បៈនៃប្រាជ្ញាគឺជាសិល្បៈនៃការស្គាល់ពីអ្វីដែលត្រូវមើលរំលង។"', // This is the line printed in the homepage and console 'view-source'
+    'settings.locale' => 'en',
```
Comment (eldadfux) [inline app/config/locales/km.php]: Should be 'km'

File: app/config/locales/km.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'settings.direction' => 'ltr',
+
+    // Service - Users
+    'account.emails.team' => 'ក្រុម %s',
+    'account.emails.verification.title' => 'ការផ្ទៀងផ្ទាត់គណនី',
+    'account.emails.verification.body' => 'en.email.auth.confirm.tpl',
```
Comment (eldadfux) [inline app/config/locales/km.php]: Should be km.email.auth.confirm.tpl

File: app/config/locales/km.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    'account.emails.recovery.title' => 'កំណត់ពាក្យសម្ងាត់ឡើងវិញ',
+    'account.emails.recovery.body' => 'en.email.auth.recovery.tpl',
```
Comment (eldadfux) [inline app/config/locales/km.php]: Should be km.email.auth.recovery.tpl

File: app/config/locales/km.php [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    'account.emails.invitation.title' => 'ការអញ្ជើញចូលក្នុងក្រុម %s នៅ %s',
+    'account.emails.invitation.body' => 'en.email.auth.invitation.tpl',
```
Comment (eldadfux) [inline app/config/locales/km.php]: Should be km.email.auth.invitation.tpl

File: app/config/locales/km.php (hunk 3 above)
Comment (SuuSoJeat) [reply app/config/locales/km.php]: I got careless.

File: app/config/locales/km.php (hunk 4 above)
Comment (SuuSoJeat) [reply app/config/locales/km.php]: I got careless.

File: app/config/locales/km.php (hunk 5 above)
Comment (SuuSoJeat) [reply app/config/locales/km.php]: I got careless.

File: app/config/locales/km.php (hunk 6 above)
Comment (SuuSoJeat) [reply app/config/locales/km.php]: I got careless.

File: app/config/locales/km.php (hunk 2 above)
Comment (SuuSoJeat) [reply app/config/locales/km.php]: I got careless.

File: app/config/locales/km.php (hunk 1 above)
Comment (SuuSoJeat) [reply app/config/locales/km.php]: I got careless.

**Pull request merged**